README.md

```diff
@@ -3,7 +3,7 @@ Kashoo IAP Tester
 =================
 
 Created by Ben Kennedy on 2013-03-22.
-Copyright (c) 2013 Kashoo Inc. All rights reserved.
```

Comment:
You'll notice that I removed "All rights reserved" since we don't actually reserve the rights that are granted by the MIT license. It's also completely unnecessary to say so even if we didn't.